.github/dependabot.yml

```diff
@@ -1,7 +1,16 @@
 version: 2
+registries:
```

Comment:
that file probably needs to make it all the way to the main branch to be considered by dependabot. And if so, you probably need 2 entries, one for the main branch (without the feed), and one for the feature branch.